src/home_robot_hw/home_robot_hw/nodes/goto_controller.py

```diff
@@ -85,17 +85,18 @@ def _goal_update_callback(self, msg: Pose):
         self.goal_visualizer(pose_goal.matrix())
 
     def _enable_service(self, request):
+        self.xyt_goal = None
         self.active = True
         return TriggerResponse(
```

Comment:
this is the return type of this function - TriggerResponse - we should get in the habit of fixing these when we modify functions